extensions/adobe/experience/campaign/feedbackevent.schema.json

```diff
@@ -0,0 +1,244 @@
+              "title": "Bounce Type",
+              "type": "string",
+              "description": "Classifies whether it is a soft bounce or hard bounce."
+            },
+            "xdm:bounceClass": {
+              "title": "Bounce Class",
+              "type": "string",
+              "description": "Classification code for a given message."
+            }
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/solutionContext": {
+          "title": "Solution Context",
+          "type": "array",
+          "description": "List of solution specific key value pairs.",
+          "items": {
+            "$ref": "https://ns.adobe.com/experience/analytics/keyvalue"
+          }
+        },
+        "https://ns.adobe.com/experience/campaign/channel": {
+          "title": "Delivery channel",
+          "type": "object",
+          "properties": {
+            "xdm:type": {
+              "title": "Channel Type",
+              "type": "string",
+              "description": "Classifies whether the channel is email, push, sms."
+            },
+            "https://ns.adobe.com/experience/campaign/email": {
+              "title": "E-mail Channel",
+              "type": "object",
+              "properties": {
+                "https://ns.adobe.com/experience/campaign/interactions": {
+                  "title": "User Interactions",
+                  "type": "object",
+                  "properties": {
+                    "xdm:fbType": {
+                      "title": "FB Type",
+                      "type": "string",
+                      "description": "Type of spam report entered against this message (see RFC 5965 § 7.3)."
+                    },
+                    "xdm:reportBy": {
+                      "title": "Report By",
+                      "type": "string",
+                      "description": "E-mail Address of the entity reporting this message as spam."
+                    },
+                    "xdm:reportTo": {
+                      "title": "Report To",
+                      "type": "string",
+                      "description": "E-mail Address to which this spam report is to be delivered."
+                    },
+                    "xdm:userAgent": {
+                      "title": "User Agent",
+                      "type": "string",
+                      "description": "Value of the browser's User-Agent header."
+                    }
+                  }
+                },
+                "https://ns.adobe.com/experience/campaign/operationalContext": {
+                  "title": "Operational Context",
+                  "type": "object",
+                  "properties": {
+                    "xdm:outboundIP": {
+                      "title": "IP Address",
+                      "type": "string",
+                      "description": "Outbound IP address of the customer used to deliver the message."
+                    },
+                    "xdm:deliveryMethod": {
```

Comment:
can we move  "deliveryMethod" as "method"  part of "delivery object" at line 28?

Reply:
Here "deliveryMethod" captures the protocol by which the email message is delivered. E.g. smtp or http. Since, this field is specific to the email channel, we have kept it under "https://ns.adobe.com/experience/campaign/email ->
https://ns.adobe.com/experience/campaign/operationalContext".